services/status-api/openapi.yml

```diff
@@ -0,0 +1,103 @@
+openapi: '3.0.3'
+info:
+  title: Status
+  description: Api for fetching api status messages.
+  contact:
+    name: GitHub Repo
+    url: https://github.com/helsingborg-stad/helsingborg-io-sls-api
+  version: '1.0'
+  license:
+    name: MIT
+    url: 'https://github.com/helsingborg-stad/helsingborg-io-sls-api#license'
+servers:
+  - url: https://dev.api.helsingborg.io
+  - url: https://release.api.helsingborg.io
+  - url: https://api.helsingborg.io
+
+paths:
+  /status/messages:
+    get:
+      tags:
+        - Status
+      summary: Get API status messages.
+      description: |
+        This endpoint returns a list of messages that would be used and displayed by a consumer.
+        The list of messages returned in the response could be on of the following:
```

Comment:
Should refrain from using suggestive actions I think, so skip the "that would be used..." part and simply just explain how this endpoint works (inputs and outputs).

Could also probably rephrase this to something like "in order of severity:".